### DISCUSSION TITLE
Pi Zero 2

### DISCUSSION THREAD

**Discussion by thumDer** [opencardev/crankshaft#510] [General]
Has anyone experience with the new new Zero 2 W? I understand that the original is not able to run crankshaft, but since the new one got much faster and has quad-core cpu I've just started wondering if it could be enough. Any thoughts or experiences?

**Comment by matt2005** [comment]
I plan on testing once i can get hold of one. It may work straight away as the Pi Zero 2 is similar to Pi 3

**Comment by matt2005** [comment]
Does anyone have a pi zero 2 that can try the latest build?

**Comment by ertugrul013** [comment]
Will try to take a look later this week

**Comment by matt2005** [comment]
My pizero 2 should arrive soon😁

**Comment by ertugrul013** [comment]
I have tested it out it looks like it works out of the box. Haven't looked to much into it
